projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.scss

```diff
@@ -1,7 +1,13 @@
 @import 'font';
 
 .log-events-table {
-  margin-top: 25px;
+  padding-top: 25px;
```

Comment:
we use spacing in multiple of 4. what does the mock say here? perhaps 24 ?

Reply:
Yeah,Fixed!!